### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by paimaam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,60 @@ A dotnet client library for the KMD Logic SMS, which allows to create a provider
...
 
 ```csharp
-IKMDLogicSMSServiceAPI kMDLogicSMSServiceAPI;
+var credentials  = new TokenCredentials(token);
+var client = new KMDLogicSMSServiceAPI(credentials)
+
+\\Creating Provider Configuration using various providers
+
+\\Link Mobility
+var result = client.CreateLinkMobilityProviderConfiguration(
```
Comment (Araballi) [inline README.md]: Can we have different code snippets  for each provider

File: README.md [hunk 2]
```diff
@@ -4,16 +4,60 @@ A dotnet client library for the KMD Logic SMS, which allows to create a provider
...
 
-Use the `IKMDLogicSMSServiceAPI` like this 
+Use the `KMDLogicSMSServiceAPI` like this 
 
 ```csharp
-IKMDLogicSMSServiceAPI kMDLogicSMSServiceAPI;
+var credentials  = new TokenCredentials(token);
+var client = new KMDLogicSMSServiceAPI(credentials)
+
+\\Creating Provider Configuration using various providers
```
Comment (Araballi) [inline README.md]: remove from code snippet and explain as text

File: README.md [hunk 3]
```diff
@@ -4,16 +4,60 @@ A dotnet client library for the KMD Logic SMS, which allows to create a provider
 ## How to use this client library
 In projects or components integrate **Logic sms service** by adding a reference to nuget package [Kmd.Logic.Sms.Client](https://www.nuget.org/packages?q=Kmd.Logic.Sms.Client)
 
-Use the `IKMDLogicSMSServiceAPI` like this 
+Use the `KMDLogicSMSServiceAPI` like this
```
Comment (Araballi) [inline README.md]: I think it is Interface

File: README.md (hunk 1 above)
Comment (paimaam) [reply README.md]: resolved

**Pull request merged**